hackathon_site/dashboard/frontend/src/components/dashboard/ItemTable/ItemTable.js

```diff
@@ -95,7 +95,7 @@ export const UnconnectedCheckedOutTable = ({
                         <TableHead>
                             <TableRow>
                                 <TableCell className={styles.widthFixed} />
```

Comment:
Didn't get through in my last review somehow, but this is undefined (because it was removed from the stylesheet)